test/UnitTests/MSTest.Core.Unit.Tests/Attributes/DynamicDataAttributeTests.cs

```diff
@@ -463,7 +442,6 @@ public void TestMethod4()
     /// <summary>
     /// The test method 5.
     /// </summary>
-    [FrameworkV2::Microsoft.VisualStudio.TestTools.UnitTesting.TestMethodAttribute]
```

Comment:
This is part of a `DummyClass`, I am not sure that we don't actually expect this to be a `TestMethod` for tests to work properly.